microsoftgraph/microsoft-graph-explorer-v4
# Fix: Add focus on Request body editor

## Overview

Fix #1858 

## Testing Instructions

1. Open the above Graph Explorer in Edge browser.
2. Navigate to the Request body edit box using tab key.
3. Observe the fix.
